recipes/scrapy/meta.yaml

```diff
@@ -0,0 +1,100 @@
+{% set name = "Scrapy" %}
+{% set version = "1.1.3" %}
+{% set hash_type = "md5" %}
+{% set hash_val = "9e5fb4f7933f4152288b45d22bb7b118" %}
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  {{ hash_type }}: {{ hash_val }}
+
+build:
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+  entry_points:
+    - scrapy = scrapy.cmdline:execute
+  number: 0
+
+# The requirements below shall match the requirements from the target package
+# version (check setup.py for changes).
+requirements:
+  build:
+    - python x.x
+    - setuptools
+    - twisted >=10.0.0
+    - w3lib >=1.15.0
+    - queuelib
+    - lxml
+    - pyopenssl
+    - cssselect >=0.9
+    - six >=1.5.2
+    - parsel >=0.9.3
+    - pydispatcher >=2.0.5
+    - service_identity
+    - pywin32  # [win]
+
+  run:
+    - python x.x
+    - twisted >=10.0.0
+    - w3lib >=1.15.0
+    - queuelib
+    - lxml
+    - pyopenssl
+    - cssselect >=0.9
+    - six >=1.5.2
+    - parsel >=0.9.3
+    - pydispatcher >=2.0.5
+    - service_identity
+    - pywin32  # [win]
+
+test:
+  imports:
+    - scrapy
+    - scrapy.commands
+    - scrapy.contracts
+    - scrapy.contrib
+    - scrapy.contrib.downloadermiddleware
+    - scrapy.contrib.exporter
+    - scrapy.contrib.linkextractors
+    - scrapy.contrib.loader
+    - scrapy.contrib.pipeline
+    - scrapy.contrib.spidermiddleware
+    - scrapy.contrib.spiders
+    - scrapy.contrib_exp
+    - scrapy.contrib_exp.downloadermiddleware
+    - scrapy.core
+    - scrapy.core.downloader
+    - scrapy.core.downloader.handlers
+    - scrapy.downloadermiddlewares
+    - scrapy.extensions
+    - scrapy.http
+    - scrapy.http.request
+    - scrapy.http.response
+    - scrapy.linkextractors
+    - scrapy.loader
+    - scrapy.pipelines
+    - scrapy.selector
+    - scrapy.settings
+    - scrapy.spidermiddlewares
+    - scrapy.spiders
+    - scrapy.utils
+    - scrapy.xlib
+    - scrapy.xlib.tx
+
+  commands:
+    # Ideally, we will run tests here but we require to provide the tests
+    # requirements first.
+    - scrapy version
+
+about:
```

Comment:
Please add:

- `license_file: LICENSE`
- `dev_url: https://github.com/scrapy/scrapy`
- `doc_url: https://scrapy.org/doc/`

and possibly a more detailed description. See for example the recipe that Continuum uses in the package that's available through the default channel:

https://github.com/ContinuumIO/anaconda-recipes/blob/master/scrapy/meta.yaml#L38-L47